util/find-uuid-in-uri.ts

```diff
@@ -0,0 +1,10 @@
+import { v4 as uuidv4 } from 'uuid';
+
+export function getIdentifierFromPersonUri(uri: string) {
+  const personBaseUri = 'http://data.lblod.info/id/personen/';
+  if (!uri.includes(personBaseUri)) {
+    return uuidv4();
+  }
+
+  return uri.replace(personBaseUri, '').trim();
+}
```

Comment:
Updated the get uuid here

Reply:
![image](https://github.com/lblod/mandataris-service/assets/36266973/37c0f0d4-6072-4d00-8d4a-0533bc1ad70b)